chef/automate
# Missing Cookbook suggestions

### :nut_and_bolt: Description
When there are over 105 cookbooks (or any other array value) on one node the last cookbooks in alphabetic order are not added to the suggestions list. This prevents a user from filter from these missing cookbooks. 

### :athletic_shoe: Demo Script / Repro Steps
1. Run automate with the current code `start_all_services`
1. `send_chef_run_example`
1. Open https://a2-dev.test/client-runs
1. Select the node and open the "Attributes" section. 
1. Select the "Default" filter and write down an attribute that is the closest to the end of the alphabetic that you can find (I found "yum").
1. Go back to the node list page (https://a2-dev.test/client-runs)
1. In the search bar select "Attribute" and type in the attribute, you found in the above step. 
1. No attributes should not be displayed in the suggestions. 
1. Run with the new code `rebuild components/config-mgmt-service`
1. On the node list page (https://a2-dev.test/client-runs) again type in the attribute, you found above and it should appear in the suggestions. 

### :chains: Related Resources

https://github.com/chef/automate/issues/690

### :white_check_mark: Checklist
